platform-includes/performance/configure-sample-rate/javascript.cloudflare.mdx

```diff
@@ -0,0 +1,65 @@
+<SignInNote />
+
+```javascript
+import * as Sentry from "@sentry/bun";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [],
+});
+```
+
+## Cloudflare Pages
+
+To use this SDK, add the `sentryPagesPlugin` as
+[middleware to your Cloudflare Pages application](https://developers.cloudflare.com/pages/functions/middleware/).
+
+We recommend adding a `functions/_middleware.js` for the middleware setup so that Sentry is initialized for your entire
+app.
+
+```javascript
+// functions/_middleware.js
```

Comment:
Same suggestion as above, can we move that into the `{ filename }` directive instead?